packages/cli/src/cli/index.ts

```diff
@@ -196,6 +199,15 @@ const main = async () => {
       );
     });
 
+  program
+    .command("install [projectPath]")
```

Comment:
ideally we want to also allow passing `--nightly` (installs `@nightly` version of packages) and `--dev` (installs `@dev` version of packages)